azkaban-common/src/main/java/io/kubernetes/autoscaling/models/V1beta2VerticalPodAutoscalerStatusRecommendation.java

```diff
@@ -0,0 +1,104 @@
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaClientCodegen", date = "2022-10-02T17:31:04.907Z[Etc/UTC]")
+public class V1beta2VerticalPodAutoscalerStatusRecommendation {
+  public static final String SERIALIZED_NAME_CONTAINER_RECOMMENDATIONS = "containerRecommendations";
+  @SerializedName(SERIALIZED_NAME_CONTAINER_RECOMMENDATIONS)
+  private List<V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations> containerRecommendations = null;
+
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendation containerRecommendations(List<V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations> containerRecommendations) {
+
+    this.containerRecommendations = containerRecommendations;
+    return this;
+  }
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendation addContainerRecommendationsItem(V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations containerRecommendationsItem) {
+    if (this.containerRecommendations == null) {
+      this.containerRecommendations = new ArrayList<>();
+    }
+    this.containerRecommendations.add(containerRecommendationsItem);
+    return this;
+  }
+
+   /**
+   * Resources recommended by the autoscaler for each container.
+   * @return containerRecommendations
+  **/
+  @javax.annotation.Nullable
+  @ApiModelProperty(value = "Resources recommended by the autoscaler for each container.")
+
+  public List<V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations> getContainerRecommendations() {
+    return containerRecommendations;
+  }
+
+
+  public void setContainerRecommendations(List<V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations> containerRecommendations) {
+    this.containerRecommendations = containerRecommendations;
+  }
+
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    V1beta2VerticalPodAutoscalerStatusRecommendation v1beta2VerticalPodAutoscalerStatusRecommendation = (V1beta2VerticalPodAutoscalerStatusRecommendation) o;
+    return Objects.equals(this.containerRecommendations, v1beta2VerticalPodAutoscalerStatusRecommendation.containerRecommendations);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(containerRecommendations);
+  }
+
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("class V1beta2VerticalPodAutoscalerStatusRecommendation {\n");
+    sb.append("    containerRecommendations: ").append(toIndentedString(containerRecommendations)).append("\n");
+    sb.append("}");
+    return sb.toString();
+  }
+
+  /**
+   * Convert the given object to string with each line indented by 4 spaces
+   * (except the first line).
+   */
+  private String toIndentedString(Object o) {
```

Comment:
If we're going to use these methods several places, can we put them in a utility?

Reply:
See the description. The first commit can be ignored. Those VPA client code are auto-generated .